src/pyfme/aircrafts/cessna_172.py

```diff
@@ -241,13 +248,13 @@ def _calculate_aero_lon_forces_moments_coeffs(self):
         )
         # FIXME: CM_q multiplicado por 2 hasta que alpha_dot pueda ser calculado
 
-    def _calculate_aero_lat_forces_moments_coeffs(self):
+    def _calculate_aero_lat_forces_moments_coeffs(self, system):
         delta_aile = np.rad2deg(self.controls['delta_aileron'])  # deg

```

Comment:
Same as above: perhaps instead of using the "internal" dictionary, the `@property`es could be used.